src/content/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data.mdx

```diff
@@ -82,35 +82,28 @@ When reporting custom events and attributes, follow these general requirements f
         * For events sent via [Event API](/docs/insights/insights-data-sources/custom-data/send-custom-events-event-api): 4096 characters
         * For events sent via agents: 255 characters
 
-          Maximum total attributes per event: 254. **Exception:** If you use an APM agent API, the max is 64.
+        Maximum total attributes per event: 254. **Exception:** If you use an APM agent API, the max is 64.
 
-          Maximum total attributes per event type: 48,000.
+        Maximum total attributes per event type: 48,000.
 
-          <Callout variant="important">
-            Charts may only display the first 255 characters of attribute values. For complete attribute values, use the [JSON chart type](/docs/insights/use-insights-ui/manage-dashboards/chart-types#widget-json) or [Query API](/docs/insights/insights-api/get-data/query-insights-event-data-api).
-          </Callout>
+        <Callout variant="important">
+          Charts may only display the first 255 characters of attribute values. For complete attribute values, use the [JSON chart type](/docs/insights/use-insights-ui/manage-dashboards/chart-types#widget-json) or [Query API](/docs/insights/insights-api/get-data/query-insights-event-data-api).
+        </Callout>
       </td>
     </tr>
 
     <tr>
       <td>
-        Naming syntax
+        Naming syntax for events and attributes
       </td>
 
       <td>
-        Attribute names can be a combination of alphanumeric characters, colons (`:`), periods (`.`), and underscores (`_`).
+        Attribute names can be a combination of alphanumeric characters, periods (`.`), and underscores (`_`), while event names can also include colons (`:`). We recommend starting the name with a letter: if it starts with something else, you'll need to put backticks around the name when querying. For more on when backticks are required in a query, see [NRQL reference](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/#syntax).
```

Comment:
I think something got garbled in the multiple edits.
Attribute names can include alphanumerics, colons, periods, and underscores.
Event names can include alphanumerics, colons, and underscores.

Reply:
ok thanks. i think i got confused by the change tracking and read it wrong again.